adafruit_mcp2515/canio/__init__.py

```diff
@@ -67,6 +67,13 @@ def __init__(self, id: int, length: int, *, extended: bool = False):
         self.extended = extended
 
 
+# Replace the above implementation with core canio implementation if it is available
```

Comment:
What do you think about moving the above into a separate file and then importing it if canio fails? That'd simplify this file and make it faster to load when canio is available.